src/doc/doc.pcss

```diff
@@ -62,37 +60,49 @@
 .doc > p code,
 .doc > table code {
   font-family: Monaco, Menlo, "Fira Mono", monospace;
-  font-size: 0.9rem;
+  font-size: 0.825rem;
+}
+
+td > a,
```

Comment:
Selector without a prefix is unsafe. It could break CSS in some other part of website

Reply:
thx, added.